src/schema/rules/dataset_metadata.yaml

```
@@ -0,0 +1,53 @@
+dataset_description:
+    selectors:
+    - path == /dataset_description.json
```

We should probably quote these things.

```suggestion
    - path == "/dataset_description.json"
```